hack/update-deps.sh

```diff
@@ -12,28 +12,40 @@ set -o pipefail
 
 cd ${ROOT_DIR}
 
+# This controls the release branch we track.
+VERSION="master"
+
+# We need these flags for things to work properly.
+export GO111MODULE=on
+export GOFLAGS=-mod=vendor
+
 # The list of dependencies that we track at HEAD and periodically
 # float forward in this repository.
 FLOATING_DEPS=(
-  "knative.dev/pkg"
-  "knative.dev/eventing"
-  "knative.dev/test-infra"
+  "knative.dev/pkg@{$VERSION}"
+  "knative.dev/eventing@{$VERSION}"
+  "knative.dev/test-infra@{$VERSION}"
```

Comment:
These are incorrect, the `$` goes outside of the curly braces.  It broke in sample-controller this morning.